### PR TITLE
Update package baseline

### PR DESCRIPTION
**Pull request by wtgodbe**
Contributes to https://github.com/aspnet/AspNetCore/issues/12702

The update to `Baseline.xml` is manual - I added all packages that AspNetCore shipped in 3.0 GA, but picked only 1 version of the shared framework package (`Microsoft.AspNetCore.App.runtime-winx64`). The update to `Basline.Designer.props` was done by running the BaselineGenerator project.

@anurse @dougbu who else should review this change?

### CODE REVIEW

**Comment by wtgodbe** [comment]
> F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'Microsoft.AspNetCore.Components.Analyzers' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]
F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'Microsoft.AspNetCore.Authorization' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]
F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'Microsoft.JSInterop' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]
F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'Microsoft.AspNetCore.Components.Analyzers' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]
F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'Microsoft.AspNetCore.Authorization' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]
F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'Microsoft.JSInterop' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]
F:\workspace.1\_work\1\s\eng\targets\ResolveReferences.targets(199,5): error BUILD001: Reference to 'System.ComponentModel.Annotations' was removed since the last stable release of this package. This could be a breaking change. See docs/ReferenceResolution.md for instructions on how to update changes to references or suppress this warning if the error was intentional. [F:\workspace.1\_work\1\s\src\Components\Components\src\Microsoft.AspNetCore.Components.csproj]

**Comment by wtgodbe** [comment]
The error is about dependencies in the .nuspec: https://github.com/aspnet/AspNetCore/blob/master/src/Components/Components/src/Microsoft.AspNetCore.Components.csproj#L33-L34, https://github.com/aspnet/AspNetCore/blob/master/src/Components/Components/src/Microsoft.AspNetCore.Components.csproj#L46-L47, https://github.com/aspnet/AspNetCore/blob/release/3.0/src/Components/Components/src/Microsoft.AspNetCore.Components.multitarget.nuspec#L6-L15. Should I add a `SuppressBaselineReference` for these 4 to the `.csproj` ?

**Comment by anurse** [comment]
It looks like this is missing all the shared framework assemblies. It only seems to have the NuGet packages. That may also be related to your errors. In general, we shouldn't see many (if any) **removed** IDs from the 2.2 list, only additions and the version changes. There may be some removals, but not as many as I see in the diff right now.

**Comment by wtgodbe** [comment]
@anurse I initally had the shared framework assemblies in `baseline.xml`, but the BaselineGenerator would fail since it was trying to download a package corresponding to each entry in the xml file. The current `basline.xml` only seems to reference packages: https://github.com/aspnet/AspNetCore/blob/master/eng/Baseline.xml

**Comment by anurse** [comment]
@wtgodbe fair enough. It's possible we need to update the generator though since we do still care about monitoring dependencies of the shared framework packages. That may be beyond the scope right now.

**Comment by wtgodbe** [comment]
@anurse in 2.x did we ship standalone packages for the assemblies currently in the shared framework? If so we should probably file a separate issue to modify the baselineGenerator to account for the shared framework package

**Comment by anurse** [comment]
> in 2.x did we ship standalone packages for the assemblies currently in the shared framework?

Yes. As part of supporting .NET Framework we had packages for every assembly, even though they were also in the shared framework. That's probably why this process was "good enough" for 2.x, but now we have stopped shipping packages (since we dropped support for running ASP.NET Core on .NET Framework).

**Comment by wtgodbe** [comment]
So for now, is the right fix to add `SuppressBaselineReference ` as mentioned in https://github.com/aspnet/AspNetCore/pull/14437#issuecomment-535194637

**Comment by wtgodbe** [comment]
I suppressed the errors with https://github.com/aspnet/AspNetCore/pull/14437/commits/e233890a257b5a20b1c9b300a7a37dff92acfec0, @dougbu @anurse please comment on if that's a sufficient fix for now. I'll also open an issue to update PackageBaselineGenerator once this is in a good state.

**Comment by wtgodbe** [comment]
Windows failures are due to `TAR` not being on the machines. At least we got past the MSBuild thing!

**Comment by wtgodbe** [comment]
> F:\workspace\_work\1\s\src\Framework\ref\Microsoft.AspNetCore.App.Ref.csproj(177,5): error MSB3073: The command "tar -czf F:\workspace\_work\1\s\artifacts\installers\Release\aspnetcore-targeting-pack-3.0.0.tar.gz ." exited with code 9009.

File: src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj [hunk 1]
```diff
@@ -154,6 +154,11 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <CreateDirectory Include="$(LocalInstallationOutputPath)" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <TarCommand Condition="'$(OS)' == 'Windows_NT'">C:\Windows\System32\tar</TarCommand>
```
Comment (dougbu) [inline src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj]:
```suggestion
    <TarCommand Condition="'$(OS)' == 'Windows_NT'">C:\Windows\System32\tar.exe</TarCommand>
```

But, I'm doubtful this is going to solve the problem. May need to use `C:\Program Files\Git\usr\bin\tar.exe` or a similar path, depending on where and how `git` is installed.

**Comment by wtgodbe** [comment]
Closing in favor of https://github.com/aspnet/AspNetCore/pull/14534 (git weirdness)

**Pull request closed without merging**